scripts/error-codes/codes.json

```diff
@@ -143,58 +143,59 @@
   "141": "Expected onSetChildren() to fire for a container child before its parent includes it in onSetChildren().",
   "142": "Expected onBeforeMountComponent() parent and onSetChildren() to be consistent (%s has parents %s and %s).",
   "143": "React.Children.only expected to receive a single React element child.",
-  "144": "React.PropTypes type checking code is stripped in production.",
```

Comment:
This code was added in 673f514 for an alpha release and is no longer around (apparently).